### PR TITLE
Fix canonical & social image URL paths for static experiment pages

### PR DESCRIPTION
**Pull request by lmorchard**
- Point canonical header link to prod experiment page path
- Un-escape canonical and social image paths

### CODE REVIEW

**Comment by lmorchard** [comment]
This doesn't help us for #1434, but I think it does fix an issue I think I see on testpilot-static.dev.mozaws.net in trying to validate the sharing headers. In particular:
- Facebook sees the canonical URL pointed to the home page, and uses that for the sharing card instead of the experiment page.
- Twitter sees the HTML-encoded URL path and doesn't recognize it as an image.

File: frontend/src/templates/index.mustache [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <meta name="viewport" content="width=device-width">
     <link rel="localization" href="/static/locales/{locale}/app.ftl">
 
-    <link rel="canonical" href="https://testpilot.firefox.com/">
+    <link rel="canonical" href="https://testpilot.firefox.com/{{{ canonical_path }}}">
```
Comment (dannycoates) [inline frontend/src/templates/index.mustache:14]: Locally, when I open the file from the filesystem I see the correct path here, but if I curl (for example) `http://testpilot.dev:8000/experiments/min-vid` I only see `https://testpilot.firefox.com/`. I can't tell if that's a side effect of the dev server or not.

Comment (lmorchard) [reply frontend/src/templates/index.mustache:14]: Yeah, I think that's an artifact of the [connect-history-api-fallback](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/server.js#L13) module showing the root `index.html`. If you add a trailing slash, you should see something different - i.e. `min-vid/`

Comment (dannycoates) [reply frontend/src/templates/index.mustache:14]: even with the trailing slash I'm not seeing the full path here, but it seems likely its just a quirk of the local server.

**Review by dannycoates — APPROVED** [review]
LGTM

**Pull request merged**